session.go

```diff
@@ -61,6 +67,13 @@ func (c *Config) name() string {
 	return c.Name
 }
 
+func (c *Config) sameSite() http.SameSite {
+	if c.SameSite == 0 {
+		return http.SameSiteDefaultMode
```

Comment:
SameSiteDefaultMode produces the [same behavior](https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/net/http/cookie.go;l=229-230) as the zero value, so we don't need this check. I think it's fine for the callsite to use c.SameSite directly.